### DISCUSSION TITLE
Send help to my poor CPU

### DISCUSSION THREAD

**Discussion by MrAnonymous722** [blakeblackshear/frigate#1801] [Show and tell]
While I wait on my Coral device to ship, this is what my current machine is dealing with haha

![image](https://user-images.githubusercontent.com/26932670/133896295-f117839d-5911-49bf-aafa-69eae801b634.png)
![image](https://user-images.githubusercontent.com/26932670/133896303-87a58e18-0f8e-46bd-b9ab-431d70d155d0.png)
![image](https://user-images.githubusercontent.com/26932670/133896308-4061d7ae-c952-48a8-97db-d2d2806d2844.png)

I think a gust of wind will set this chain reaction of queued things to detect and it just gets worse 'till it gets still enough to catch up.

**Comment by blakeblackshear** [comment]
You can try using a smaller frame_height under motion. This should require a larger area of motion to run detection. 
https://github.com/blakeblackshear/frigate/blob/release-0.9.0/docs/docs/configuration/index.md#full-configuration-reference
